### DISCUSSION TITLE
How i can deployed flet web application on my linux server?

### DISCUSSION THREAD

**Discussion by vist-amur** [flet-dev/flet#802] [Q&A]
Hi, guys!
How i can deployed flet web application on my linux server?  Is it possible?

**Comment by jet10000** [comment]
Yes, of course

**Reply by vist-amur** [reply]
Could you tell, how this do right, please?

**Comment by vist-amur** [comment]
Could you tell, how this do right, please?

**Comment by vist-amur** [comment]
Help me deploy app on linux server, please! Very need!
